schema/tokite_rules.schema

```diff
@@ -5,6 +5,7 @@ create_table "tokite_rules", force: :cascade do |t|
   t.string   "channel",         limit: 100,  null: false
   t.string   "icon_emoji",      limit: 20,   null: false, default: ""
   t.string   "additional_text", limit: 200,  null: false, default: ""
+  t.string   "display_name",    limit: 60,   null: false, default: ""
```

Comment:
display_name must be added to the last.
https://github.com/cookpad/tokite/pull/41#discussion_r195951523

Reply:
Thanks, I forgot it.